server/src/main/java/com/saralein/server/HttpServer.java

```diff
@@ -0,0 +1,45 @@
+package com.saralein.server;
+
+import com.saralein.server.controller.ErrorController;
+import com.saralein.server.controller.DirectoryController;
+import com.saralein.server.controller.FileController;
+import com.saralein.server.logger.Logger;
+import com.saralein.server.request.RequestParser;
+import com.saralein.server.response.ResponseSerializer;
+import com.saralein.server.router.Routes;
+import com.saralein.server.router.Router;
+import com.saralein.server.setup.Server;
```

Comment:
Why is `Server` in the `com.saralein.server.setup` package. It feels like it would belong in `com.saralein.server`